src/components/team/billing-section.tsx

```diff
@@ -19,18 +18,26 @@ const formatAmountWithCurrency = (
   }).format(amountInCents / 100)
 }
 
+const recur = (price: any) => {
+  if (price === undefined) return ''
+
+  const {
+    recurring: {interval, interval_count},
+  } = price
+
+  if (interval === 'month' && interval_count === 3) return 'quarter'
+  if (interval === 'month' && interval_count === 6) return '6-months'
+  if (interval === 'month' && interval_count === 1) return 'month'
+  if (interval === 'year' && interval_count === 1) return 'year'
+}
+
```

Comment:
I pulled this function out of the subscription data hook and placed it here because it was only being used here.